packages/odyssey-react-mui/src/themes/odyssey/components.ts

```diff
@@ -399,6 +417,29 @@ export const components: ThemeOptions["components"] = {
       },
     },
   },
+  MuiInputLabel: {
+    defaultProps: {
+      disableAnimation: true,
+      shrink: false,
+    },
+    styleOverrides: {
+      root: ({ ownerState }) => ({
+        // @ts-expect-error: Incorrect typing in MUI
+        ...(ownerState.formControl && {
+          position: "initial",
+          transform: "none",
+        }),
+        ...(ownerState.variant === "outlined" && {
+          pointerEvents: "initial",
+          transform: "none",
+          maxWidth: "100%",
+          ...(ownerState.size === "small" && {
+            transform: "none",
+          }),
+        }),
+      }),
+    },
+  },
```

Comment:
I'd like to look into making these `styleOverrides` as a template string or an Emotion CSS template string:
```js
styleOverrides: {
  root: ({ ownerState }) => css`
    ${ownerState.formControl && css`
      position: initial;
      transform: none;
    `}
  `
}
```

We might not need the `css` template string, but you'd typically `import { css } from '@emotion/react'`.

Reply:
@magizh-okta this might be worth a spike. I'm not strongly opinionated here -- happy to do whatever feels most natural to devs as long as we don't lose any MUI sugar.